taf/git.py

```diff
@@ -28,6 +28,9 @@
 
 EMPTY_TREE = "4b825dc642cb6eb9a060e54bf8d69288fbee4904"
 
+class PygitError(GitError):
```

Comment:
This should be in `exceptions.py`, you'll see that that is where all of the custom errors are